packages/shared/prisma/schema.prisma

```diff
@@ -675,8 +675,8 @@ model PromptDependency {
   childLabel   String? @map("child_label")
   childVersion Int?    @map("child_version")
 
-  @@index([projectId, parentId])
-  @@index([projectId, childName])
+  @@index([projectId, parentId], map: "prompt_dependencies_project_id_parent_id")
+  @@index([projectId, childName], map: "prompt_dependencies_project_id_child_name")
```

Comment:
@hassiebp FYI: Somehow your names didn't fit the prisma auto-generated ones. I had to add this overwrite to avoid another modification in my SQL.